src/pages/ServiceDetails/ServiceDetailsPage.tsx

```diff
@@ -12,9 +12,9 @@ import { DurationInSeconds } from '../../types/Common';
 import { durationSelector } from '../../store/Selectors';
 import ParameterizedTabs, { activeTab } from '../../components/Tab/Tabs';
 import ServiceInfo from './ServiceInfo';
-import TrafficDetails from '../../components/Metrics/TrafficDetails';
 import TracesComponent from 'components/JaegerIntegration/TracesComponent';
 import { JaegerInfo } from 'types/JaegerInfo';
+import TrafficDetails from 'pages/TrafficList/TrafficDetails';
```

Comment:
TrafficList is not a page, it's a component.
Specific components used for a page can go inside.
Shared components across all pages should go as a component.